x-pack/plugins/apm/server/routes/metrics/get_metrics_chart_data_by_agent.ts

```diff
@@ -42,13 +45,15 @@ export async function getMetricsChartDataByAgent({
     start,
     end,
   };
+  const isOpenTelemetry = isOpenTelemetryAgentName(agentName);
```

Comment:
Could we make the decision which metrics to show based on the available metrics rather than the agent name? This would allow us to align with the OTel metrics format in the Java agent in the future.